.github/actions/spelling/line_forbidden.patterns

```diff
@@ -0,0 +1,96 @@
+# reject `m_data` as VxWorks defined it and that breaks things if it's used elsewhere
+# see [fprime](https://github.com/nasa/fprime/commit/d589f0a25c59ea9a800d851ea84c2f5df02fb529)
+# and [Qt](https://github.com/qtproject/qt-solutions/blame/fb7bc42bfcc578ff3fa3b9ca21a41e96eb37c1c7/qtscriptclassic/src/qscriptbuffer_p.h#L46)
+# \bm_data\b
+
+# If you have a framework that uses `it()` for testing and `fit()` for debugging a specific test,
+# you might not want to check in code where you were debugging w/ `fit()`, in which case, you might want
+# to use this:
+#\bfit\(
+
+# s.b. anymore
+\bany more[,.]
+
+# s.b. GitHub
+(?<![&*.]|// |type )\bGithub\b(?![{)])
+
+# s.b. GitLab
+(?<![&*.]|// |type )\bGitlab\b(?![{)])
+
+# s.b. JavaScript
+\bJavascript\b
+
+# s.b. Microsoft
+\bMicroSoft\b
+
+# s.b. another
+\ban[- ]other\b
+
+# s.b. deprecation warning
+\b[Dd]epreciation [Ww]arnings?\b
+
+# s.b. greater than
+\bgreater then\b
+
+# s.b. in front of
+\bin from of\b
+
+# s.b. into
+# when not phrasal and when `in order to` would be wrong:
+# https://thewritepractice.com/into-vs-in-to/
+#\sin to\s
+
+# s.b. is obsolete
+\bis obsolescent\b
+
+# s.b. it's or its
+\bits['’]
+
+# s.b. opt-in
+(?<!\sfor)\sopt in\s
+
+# s.b. less than
+\bless then\b
+
+# s.b. otherwise
+\bother[- ]wise\b
+
+# s.b. nonexistent
+\bnon existing\b
+\b[Nn]o[nt][- ]existent\b
+
+# s.b. brief / details/ param / return / retval
+(?:^\s*|(?:\*|//|/*)\s+`)[\\@](?:breif|(?:detail|detials)|(?:params|prama?)|ret(?:uns?)|retvl)\b
+
+# s.b. preexisting
+[Pp]re[- ]existing
+
+# s.b. preempt
+[Pp]re[- ]empt\b
+
+# s.b. preemptively
+[Pp]re[- ]emptively
+
+# s.b. recently changed or recent changes
+[Rr]ecent changed
+
+# s.b. reentrancy
+[Rr]e[- ]entrancy
+
+# s.b. reentrant
+[Rr]e[- ]entrant
+
+# s.b. understand
+\bunder stand\b
+
+# s.b. workarounds
+\bwork[- ]arounds\b
+
+# s.b. workaround
+(?:(?:[Aa]|[Tt]he|ugly)\swork[- ]around\b|\swork[- ]around\s+for)
+
+# s.b. (coarse|fine)-grained
+\b(?:coarse|fine) grained\b
+
+# Reject duplicate words
```

Comment:
So much for allowing the sentence "Buffalo buffalo that buffalo Buffalo buffalo buffalo Buffalo buffalo."

Reply:
If you want to allow it, you can add `buffalo` to https://github.com/rancher-sandbox/rancher-desktop/blob/a6d37e30e2f2e56f061b663ee40844f8b4c49c0e/.github/actions/spelling/patterns.txt#L125

Reply:
No, it was an edge case, and it's always bugged me that I need that "that" to make the sentence work. Which reminds me of that GEB construct, "'Yields a sentence when appended to itself' yields a sentence when appended to itself" but I digress.

Reply:
Actually, you don't need `that` there:
> "Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo" is a [grammatically correct](https://en.wikipedia.org/wiki/Grammaticality) [sentence](https://en.wikipedia.org/wiki/Sentence_(linguistics)) in [English](https://en.wikipedia.org/wiki/English_Language) ...

https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo

It's a fun phrase, I play w/ it periodically :)